app/ide-desktop/eslint.config.js

```diff
@@ -0,0 +1,354 @@
+/** @file ESLint configuration file. */
+/** NOTE: The "Experimental: Use Flat Config" option must be enabled. */
+import * as path from 'node:path'
+import * as url from 'node:url'
+
+const DIR_NAME = path.dirname(url.fileURLToPath(import.meta.url))
+
+// The .d.ts is wrong; these only have default exports.
+// Other default imports are whitelisted; however these should only be used here.
+/* eslint-disable no-restricted-syntax */
```

Comment:
Why we need this?

Reply:
i've added lints to prefer namespace imports: `* as foo`. there is an explicit whitelist elsewhere but i don't think it needs to cover modules that are only used here. i can almost certainly make it clearer though